astrology.lic

```diff
@@ -35,6 +41,20 @@ class Astrology
     result.compact.select { |data| data['circle'] <= DRStats.circle }
   end
 
+  def check_astral
+    return unless DRStats.circle > 99
+    return unless @ap_training
+    return unless @ap_source
+    return unless @ap_destination
+
+    if UserVars.astral_plane_timer.nil? || Time.now - UserVars.astral_plane_timer > 3600
+      wait_for_script_to_complete('bescort', ['ways', @ap_destination])
+      wait_for_script_to_complete('bescort', ['ways', @ap_source])
+      UserVars.astral_plane_timer = Time.now
```

Comment:
In which case it'd be `UserVars.exp_timers['astral_plane']` does that make more sense?